main.go

```diff
@@ -184,13 +186,15 @@ func main() {
 	stepconf.Print(config)
 	fmt.Println()
 
-	if err := mainE(config); err != nil {
+	cmdFactory := command.NewFactory(env.NewRepository())
```

Comment:
Is this the right way to initialize a `command.Factory`?

Reply:
Yes, this is perfect. We have this abstraction to be able to mock things in tests